CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to Drone
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to Drone and its packages, which are hosted in the [Drone Organization](https://github.com/drone) on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## Code of Conduct
+
+  * Comming soon!
+
+## I don't want to read this whole thing I just have a question!!!
+
+> **Note:** [Please don't file an issue to ask a question.](http://docs.drone.io/getting-help/)
+
+## What should I know before I get started?
+
+### Drone and Plugins
+
+Drone in itself is comprised of [several projects](https://github.com/drone), [Community Plugins](https://github.com/drone-plugins) and [Public plugins](http://plugins.drone.io/)
+
+## How Can I Contribute?
+
+### Your First Code Contribution
+
+Feel free to look through the issues list and find something to start working on!
+
+#### Local development
+
+##### Linux
+
+  * Comming soon!
+
+##### MacOS
+
+  * Comming soon!
+
+
+## Styleguides
+
+### Go common code review comments
+
+ * Checkout Golangs [Common code review comments](https://github.com/golang/go/wiki/CodeReviewComments)
+
+### Git Commit Messages
+
+* Use the present tense ("Add feature" not "Added feature")
+* Limit the first line to 72 characters or less
+* Reference issues and pull requests liberally after the first line
+* When only changing documentation, include `[ci skip]` in the commit description
```

Comment:
The docs are not kept in here, and they shouldn't skip CI otherwise they won't get built and published.